docker/Dockerfile

```diff
@@ -49,3 +49,59 @@ VOLUME ["/data"]
 
 ENTRYPOINT ["/usr/local/bin/litentry-collator"]
 CMD ["--help"]
+
+FROM ubuntu:22.04 AS chain-aio
+LABEL maintainer="Trust Computing GmbH <info@litentry.com>"
+
+ARG PROFILE
+ENV NVM_DIR /opt/nvm
+
+# install netcat for healthcheck
+RUN apt-get update && \
+    apt install -yq build-essential ocaml ocamlbuild automake autoconf libtool wget \
+    python-is-python3 libssl-dev git cmake perl python3 dkms pip python3-click clang-13 \
+    lldb-13 lld-13 clangd net-tools libssl-dev libcurl4-openssl-dev protobuf-compiler \
+    libprotobuf-dev debhelper cmake reprepro unzip pkgconf libboost-dev libboost-system-dev \
+    libboost-thread-dev lsb-release libsystemd0 pkgconf jq python3-pip curl && \
+    apt-get install -yq ca-certificates && \
+    update-ca-certificates && \
+    wget -O /usr/local/bin/polkadot -q https://github.com/paritytech/polkadot/releases/download/v0.9.42/polkadot && \
+    chmod +x /usr/local/bin/polkadot && \
+    curl -fsSL https://deb.nodesource.com/setup_21.x | sudo -E bash - && \
+    apt-get update && apt-get install -y nodejs && \
+    npm install -g pnpm && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    apt-key del 23E7166788B63E1E && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    apt update && apt install -y yarn  && \
+    mkdir -p $NVM_DIR && \
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash && \
+    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" && \
+    [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion" && \
+    nvm install 18 && \
+    nvm use 18 && \
+    apt-get clean && \
+    rm -rf /var/cache/apt/lists 
+
+RUN useradd -m -u 1000 -U -s /bin/sh -d /litentry litentry && \
+	mkdir -p /opt/litentry/parachain /code/litentry-parachain && \
+	chown -R litentry:litentry /opt/litentry
```

Comment:
I think here miss to change the ownership of `/code/litentry-parachain`.

Reply:
The ownership change of /code/litentry-parachain must occur after the COPY operation. There is already a line for this after line 95.